src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -116,6 +115,18 @@
 	protected static Properties appExtSecurePropertiesFile=null;
 	protected static Properties appExtPropertiesFile=null;	
 
+	private static final HashMap<String, Integer> enrollmentsMap = new HashMap<String, Integer>(){
+		private static final long serialVersionUID = -1389517682290891890L;
+
+		{
+			put("ObserverEnrollment", 0);
+			put("StudentEnrollment", 0); //all others will be 0
+			put("TaEnrollment", 1);
+			put("TeacherEnrollment", 2);
+			put("DesignerEnrollment", 2);
+		}
+	};
+
```

Comment:
Please add comment the reason from numbering the enrollment in the way that is done here